completions/zsh/_crio

```diff
@@ -28,6 +28,7 @@ it later with **--config**. Global options will modify the output.'
         '--big-files-temporary-dir'
         '--bind-mount-prefix'
         '--blockio-config-file'
+        '--blockio-reconfigure'
```

Comment:
can we use `blockio-reload`? I think it conforms more to what we call this situation internally

Reply:
Thanks @haircommander! Fixed. I think it's better now.